src/templates/home/_comercial.html

```diff
@@ -0,0 +1,24 @@
+<h1>
+        Aides territoires s’adresse aux acteurs des territoires
+</h1>
+<p>
+    Le but de ce service est <b>d’agréger l’ensemble des aides destinées au financement et à l'ingénierie</b> de vos projets locaux.<br>
+    Le service est fonctionnel depuis 2018 et s’améliore chaque jour en <b>rendant disponible plus d’offres et en améliorant la qualité de recherche</b>.
+</p>
+<div>
+    <div class="comercial">
+        <img src="/static/icons/noun_Time_2630876.svg" alt="Gagnes du temps" />
+        <h2>Gagnez du temps</h2>
+        <p>Obtenez au <b>même endroit</b> les aides de la part de nombreux organismes</p>
+    </div>
+    <div class="comercial">
+        <img src="/static/icons/noun_List_2041127.svg" alt="Ne ratez rien" />
+        <h2>Ne ratez rien</h2>
+        <p>Retrouvez une large sélection d’aides <b>disponibles pour votre territoire</b></p>
+    </div>
+    <div class="comercial">
+        <img src="/static/icons/noun_Eye_2184513.svg" alt="Restez en veille" />
```

Comment:
Je ne sais pas d'où viennent les fichiers mais il y a un problème, ces images contiennent encore la signature « Created by alice design… ».

Par ailleurs, nous utilisons déjà le [pack d'icones FontAwesome](https://fontawesome.com/). Pour des raisons de cohésion graphique, je pense qu'il faudrait éviter d'intégrer un autre pack (ici, the noun project). Il me semble qu'il existe des équivalences dans FontAwesome qui pourraient faire l'affaire.

Par ailleurs, l'utilisation de ces icones, en CC By, nécessiteraient de mettre à jour les mentions légales.

Reply:
J'ai trouvé des équivalents avec FontAwesome merci